tests/integration/test_config.py

```diff
@@ -55,34 +78,45 @@ def _config(value):
         # Make sure to leave override-channels and the full channel URL in here.
         # This allows this command to run fully no matter what we do to channel_alias
         # and default_channels
+        cmd = [
+            "conda",
+            "install",
+            "-vvv",
+            "--override-channels",
+            "-c",
+            "https://repo.anaconda.com/pkgs/fakechannel",
+            "fakepackage",
+            "fakepackage",
+        ]
```

Comment:
That reminds me, integration tests can without issue use actual packages IMO, is it reasonable to use one from repo.anaconda.com, or even run a small test repo server during the tests?

Reply:
The reason I'm using a fake channel and a fake package here is to ensure that the index cache doesn't get populated. This ensures that requests to the server are made every single time

Reply:
Adding a conda clean --index here would work too I suppose!